test/cache/map_cache_test.dart

```diff
@@ -20,19 +20,15 @@ import 'package:quiver/cache.dart';
 
 void main() {
   group('MapCache', () {
-    MapCache<String, String> cache;
```

Comment:
Could this have just been:

```dart
var cache = MapCache<String, String>()
```

Less change in the file, still safe for nullability.

Reply:
Ah I see, just initialise it once here, to avoid the nullability concern, and leave the `setUp` code as-is to reset it?

Reply:
Yeah, I don't know what's really best. It feels about the same way of reading though, if backwards.

Reply:
Went with door #3, late initialization!